unstructured/partition/email.py

```diff
@@ -163,10 +163,10 @@ def extract_attachment_info(
             cdisp = part["content-disposition"].split(";")
             cdisp = [clean_extra_whitespace(item) for item in cdisp]
 
+            attachment_info = {}
             for item in cdisp:
-                attachment_info = {}
 
-                if item.lower() == "attachment":
+                if item.lower() == "attachment" or item.lower() == "inline":
```

Comment:
```suggestion
                if item.lower() in ("attachment", "inline"):
```
tiny nit

Reply:
I considered that myself. Thanks for that feedback. Fixed.